src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc

```diff
@@ -0,0 +1,72 @@
+= Alert notifiers for mobile devices
+Thomas Segismont
+2015-04-14
+:jbake-type: post
+:jbake-status: draft
```

Comment:
Le 14/04/2015 11:38, Heiko W. Rupp a écrit :
> Why draft?

In order to see how it looks online before publishing.